Packs/Threat_Crowd/Integrations/ThreatCrowdV2/ThreatCrowd_v2_test.py

```diff
@@ -0,0 +1,151 @@
+import json
+import io
+# from Packs.Base.Scripts.CommonServerPython.CommonServerPython import DBotScoreReliability
+# from Packs.Threat_Crowd.Integrations.ThreatCrowdV2.ThreatCrowd_v2 import Client
+from CommonServerPython import DBotScoreReliability
+from ThreatCrowd_v2 import Client
+
+
+def util_load_json(path):
+    with io.open(path, mode='r', encoding='utf-8') as f:
+        return json.loads(f.read())
+
+
+def get_key_from_test_data(key):
+    path = './test_data/response_mocks.json'
+    test_data = util_load_json(path)
+    return test_data[key]
+
+
+CLIENT = Client(
+    base_url='test_url',
+    verify=False,
+    proxy=False,
+    reliability=DBotScoreReliability.C
+)
+
+
+def test_ip_command(mocker):
+    """
+        Given:
+            - a list of ips
+
+        When:
+            - running ip command was required
+
+        Then:
+            - validates that indicator objects were created as expected
```

Comment:
same for the other tests
```suggestion
            - validates that indicator objects were created with the proper reliability, score and values.
```